develop/migration_data.rst

```diff
@@ -1,16 +1,32 @@
 .. _data_migration:
 
-Data Migration
-==============
+Migrate Data
+============
 
-Migrating from one database to another has traditionally been difficult. There are two traditional approaches: using a dump and restore that takes downtime, or updating application logic to write to two databases and then switch to the new one.
+At this time, having updated the database schema and application queries to work with Citus, you're ready for the final step. It's time to migrate data to the Citus cluster and cut over the application to its new database.
 
-Now there's an easier way. Citus Warp allows you to stream changes from a PostgreSQL source database into a :ref:`Citus Cloud <cloud_overview>` cluster as they happen. It's as if the application automatically writes to two databases rather than one, except with perfect transactional logic. Citus Warp works with Postgres versions 9.4 and above which have the `logical_decoding` plugin enabled (this is supported on Amazon RDS as long as you're at version 9.4 or higher).
+The data migration path is dependent on downtime requirements and data size, but generally falls into one of the following two categories.
 
-To do a warp, we connect the coordinator node of a Citus cluster to an existing database through VPC peering or IP white-listing, and begin replication.
+Databases under 200GB
+---------------------
 
-Using Citus Warp
-----------------
+Smaller environments that can tolerate a little downtime can use a simple pg_dump/pg_restore process.
+
+* Put the application in maintenance mode
+* Dump the old database using pg_dump
+* Disallow connections to old database
+* Import into Citus using pg_restore
+* Test application
+* Launch!
+
+Documentation request: Page detailing this process
```

Comment:
@colton-citus do you have a script that you suggest to people with the desired pg_dump arguments etc?